ntb_hw_switchtec.c

```diff
@@ -459,8 +459,9 @@ static void switchtec_ntb_part_link_speed(struct switchtec_ntb *sndev,
 	u32 pff;
 	u32 linksta;
 
-	pff = ioread32(&stdev->mmio_part_cfg_all[partition].vep_pff_inst_id);
-	if (pff == 0xFFFFFFFF) {
+	pff = ioread32(&stdev->mmio_part_cfg_all[partition].vep_pff_inst_id)
+	      & 0xFF;
```

Comment:
Nit: this line break is really ugly. Maybe just add `pff &= 0xFF;` on it's own line.

Reply:
Updated. Thanks.